tests/lib/pkgdb.sh

```diff
@@ -8,6 +8,29 @@ debian_name_package() {
         xdelta3|curl|python3-yaml|kpartx|busybox-static)
             echo "$1"
             ;;
+        *)
+            echo $1
+            ;;
+    esac
+}
+
+fedora_name_package() {
+    case "$1" in
+        xdelta3|jq|curl)
+            echo $1
+            ;;
+        python3-yaml)
+            echo "python3-yamlordereddictloader"
```

Comment:
What? This isn't right. It's `python3-PyYAML`. Also, `python3-yaml` name is provided by the `python3-PyYAML` package.